apps/frontend/package.json

```diff
@@ -56,31 +56,26 @@
     "react-dom": "^17.0.2",
     "react-fast-compare": "^3.2.0",
     "react-i18next": "^12.2.0",
-    "react-quill": "^2.0.0",
     "react-router": "^5.3.1",
     "react-router-dom": "^5.3.1",
-    "react-scripts": "^5.0.1",
     "tinymce": "^5.10.9",
     "typescript": "^4.6.4",
     "use-query-params": "^1.2.3",
     "yup": "^0.32.11"
   },
   "proxy": "http://localhost:4000",
   "scripts": {
-    "dev": "react-scripts start",
-    "debug": "react-scripts --inspect-brk start",
-    "dev:docker": "npm install && react-scripts start",
-    "build": "react-scripts --max_old_space_size=4096 build",
-    "test": "react-scripts test",
-    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
+    "dev": "vite",
+    "dev:docker": "npm install && vite",
+    "build": "vite build",
+    "preview": "vite preview",
     "lint": "tsc --noEmit && eslint . --ext .js,.jsx,.ts,.tsx --quiet",
     "lint:fix": "tsc --noEmit && eslint . --ext .js,.jsx,.ts,.tsx --fix",
     "lint:fix:bash": "eslint --fix $(git diff --name-only --diff-filter=ACMRTUXB develop | grep  -E \"(.js$|.ts$|.tsx$)\")",
-    "eject": "react-scripts eject",
     "start": "npm run dev",
```

Comment:
AFAIK we are not using the `start` script, might be a good chance to clean it up,